packages/vue-instantsearch/src/components/StateResults.vue

```diff
@@ -16,7 +16,6 @@
 <script>
 import { createSuitMixin } from '../mixins/suit';
 import { createWidgetMixin } from '../mixins/widget';
-import { _objectSpread } from '../util/polyfills';
```

Comment:
this is for babel in Vue InstantSearch, not for ie11

Reply:
I don't remember well why we resort to this, does the build fail if we use native object spreading in the codebase for Vue?

Reply:
yes, it was something along those lines, but if it works feel free to change

Reply:
Alright! I'll check this once I've merged FX-3053 into this branch.